docs/reference/auditbeat/setup-repositories.md

```diff
@@ -17,10 +17,111 @@ to sign all our packages. It is available from [https://pgp.mit.edu](https://pgp
 
 ## APT [_apt]
 
-Version 9.0.0-beta1 of Beats has not yet been released.
+To add the Beats repository for APT:
+
+1. Download and install the Public Signing Key:
+
+    ```shell
+    wget -qO - https://artifacts.elastic.co/GPG-KEY-elasticsearch | sudo apt-key add -
+    ```
+
+2. You may need to install the `apt-transport-https` package on Debian before proceeding:
+
+    ```shell
+    sudo apt-get install apt-transport-https
+    ```
+
+3. Save the repository definition to  +/etc/apt/sources.list.d/elastic-{{major-version}}.list+:
+
+    ```shell subs=true
+    echo "deb https://artifacts.elastic.co/packages/{{major-version}}/apt stable main" | sudo tee -a /etc/apt/sources.list.d/elastic-{{major-version}}.list
+    ```
+
+    :::{note}
+    The package is free to use under the Elastic license. An alternative package which contains only features that are available under the Apache 2.0 license is also available. To install it, use the following sources list:
+
+    ```shell subs=true
+    echo "deb https://artifacts.elastic.co/packages/oss-{{major-version}}/apt stable main" | sudo tee -a /etc/apt/sources.list.d/elastic-{{major-version}}.list
+    ```
+    :::
+
+    :::{warning}
+    To add the Elastic repository, make sure that you use the `echo` method  shown in the example. Do not use `add-apt-repository` because it will add a `deb-src` entry, but we do not provide a source package.
+
+    If you have added the `deb-src` entry by mistake, you will see an error like the following:
+
+        `Unable to find expected entry 'main/source/Sources' in Release file (Wrong sources.list entry or malformed file)`
+
+    Simply delete the `deb-src` entry from the `/etc/apt/sources.list` file, and the installation should work as expected.
+    :::
+
+4.  Run `apt-get update`, and the repository is ready for use. For example, you can install Auditbeat by running:
+
+    ```shell
+    sudo apt-get update && sudo apt-get install auditbeat
+    ```
+
+5. To configure auditbeat to start automatically during boot, run:
+
+    ```
+    sudo systemctl enable auditbeat
+    ```
```

Comment:
Maybe we need to add the steps to start the beat? `systemctl enable` doesn't really start it, and I don't think it's automatically started after the package installation.

I would offer the start command together with a link to the configuration reference so the user knows that they should probably configure the beat `.yml` file before starting it.

Reply:
@eedugon  That makes sense, but, just for code cleanliness, I think it we shouldn't both i) migrate the install instructions and ii) change the install instructions, in the same PR.

I can open an issue for changing these steps, and we can have that reviewed by a developer just to be sure the change is appropriate. Does that sound okay?